Exercise02/app/src/main/java/com/example/GameOfThrones/Data.java

```diff
@@ -0,0 +1,11 @@
+package com.example.GameOfThrones;
+
+public class Data {
+    public int imageId;
+    public String txt;
```

Comment:
1. In general, it is not recommended to "shorten" keywords, why txt and not text?
2. Same as the comment above, try to think about less general property name, what text is it? I can't refer to the text this class is about to hold from this general name.